libraries/shared/storage/backfill/storage_value_loader_test.go

```diff
@@ -122,15 +138,33 @@ var _ = Describe("StorageValueLoader", func() {
 		Expect(bc.BatchGetStorageAtCalls).To(ConsistOf(
 			fakes.BatchGetStorageAtCall{BlockNumber: bigIntBlockOne, Account: addressOne, Keys: []common.Hash{keyOne}},
 			fakes.BatchGetStorageAtCall{BlockNumber: bigIntBlockOne, Account: addressTwo, Keys: []common.Hash{keyTwo}},
+			fakes.BatchGetStorageAtCall{BlockNumber: bigIntBlockOne, Account: addressThree, Keys: []common.Hash{keyThree}},
 		))
 	})
 
+	It("chunks requests to avoid 413 Request Entity Too Large reply from server", func() {
+		manyKeys := make([]common.Hash, 401)
```

Comment:
Can you use the maxRequestSize variable here? In case you change the size of the chunk the test doesn't need to fail.

Reply:
Yeah I think that's a good call. It means we need to publicly export the variable but I think that's probably worth it. Alternatively we could inject the request size when we initialize the StorageValueLoader 🤔